numpy/_core/src/multiarray/dlpack.c

```diff
@@ -53,18 +83,41 @@ static void dlpack_capsule_deleter(PyObject *self) {
     }
 }
 
-/* used internally, almost identical to dlpack_capsule_deleter() */
-static void array_dlpack_internal_capsule_deleter(PyObject *self)
-{
-    /* an exception may be in-flight, we must save it in case we create another one */
-    PyObject *type, *value, *traceback;
-    PyErr_Fetch(&type, &value, &traceback);
```

Comment:
This was always unnecessary, just missed that it was also here when removing it in the version above...